src/settings.rs

```diff
@@ -1,23 +1,34 @@
+/*! Configurable knobs and their related errors
+*/
 use std::collections::HashMap;
 
 use clap::ArgAction;
 use url::Url;
 
 use crate::channel_value::ChannelValue;
 
+/// Default [`nix_package_url`](CommonSettings::nix_package_url) for Linux x86_64
 pub const NIX_X64_64_LINUX_URL: &str =
```

Comment:
Maybe something for later down the road, but maybe we could have these links be in a separate file that we `include_str!` from. That would make it easier to automatically keep these in sync with the latest Nix release (but again, probably save this for later) -- just have an action that finds the latest version and replaces the files.

Reply:
That's a great idea! Let's do it later for sure!